packages/react-vapor/src/components/title/Title.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */

```

Comment:
opsy.  

Will remove after appling all the other comments if there is some to apply

Reply:
vite le fouet :fishing_pole_and_fish:

Reply:
I'll make it easy for you
```suggestion
```